bob-common/src/node.rs

```diff
@@ -20,6 +23,8 @@ pub struct Node {
     address: String,
     index: Id,
     conn: Arc<RwLock<Option<BobClient>>>,
+    err_count: Arc<AtomicUsize>,
```

Comment:
`Node` is bad type for this counter. It is better to count errors inside `BobClient`, because it is actually executes all the operations